src/pages/createPost.tsx

```diff
@@ -154,122 +62,111 @@ const createPost = () => {
             </div>
           </Grid>
-          
           <Grid item>
-
-          <Typography variant="h5">Date time</Typography>
-          <Grid container spacing={2}>
-          <Grid item xs={6}>
-              <Typography variant="body1">Start</Typography>
-              <LocalizationProvider dateAdapter={AdapterDayjs}>
-              <DateTimePicker
-          
-          value={value}
-          onChange={handleChange}
-          renderInput={(params) => <TextField {...params} fullWidth/>}
-          
-        /></LocalizationProvider>
+            <Typography variant="h5">Date time</Typography>
+            <Grid container spacing={2}>
+              <Grid item xs={6}>
+                <Typography variant="body1">Start</Typography>
+                <LocalizationProvider dateAdapter={AdapterDayjs}>
+                  <DateTimePicker
+                    value={value}
+                    onChange={handleChange}
+                    renderInput={(params) => <TextField {...params} fullWidth />}
+                  />
+                </LocalizationProvider>
                 <div style={helperTextBox}>
                   <FormHelperText sx={helperTextError}>ช่องนี้ไม่สามารถเว้นว่างได้</FormHelperText>
                 </div>
-            
-            </Grid> 
-
-          <Grid item xs={6}>
+              </Grid>
+              <Grid item xs={6}>
                 <Typography variant="body1">End</Typography>
                 <LocalizationProvider dateAdapter={AdapterDayjs}>
-              <DateTimePicker
-          
-          value={value}
-          onChange={handleChange}
-          renderInput={(params) => <TextField {...params} fullWidth />}
-          
-        /></LocalizationProvider>
-                  <div style={helperTextBox}>
-                    <FormHelperText sx={helperTextError}>ช่องนี้ไม่สามารถเว้นว่างได้</FormHelperText>
-                  </div>
-            
-          </Grid> 
-
-        </Grid>
-
+                  <DateTimePicker
+                    value={value}
+                    onChange={handleChange}
+                    renderInput={(params) => <TextField {...params} fullWidth />}
+                  />
+                </LocalizationProvider>
+                <div style={helperTextBox}>
+                  <FormHelperText sx={helperTextError}>ช่องนี้ไม่สามารถเว้นว่างได้</FormHelperText>
+                </div>
+              </Grid>
+            </Grid>
           </Grid>
-
-
           <Grid item>
             <Typography variant="h5">Tag</Typography>
             <Autocomplete
-            multiple
-            limitTags={5}
-            id="multiple-limit-tags"
-            options={Tags}
-            getOptionLabel={(option) => option.Tag}
-            renderInput={(params) => (
-              <TextField {...params} placeholder="Click for add tag" fullWidth />
-            )}
-            
+              multiple
+              limitTags={5}
```

Comment:
When the number of tags is more than `limitTags`, It become +1, +2. and still allow user to add more tags instead of disable